assets/schema/ensemble_schema.json

```diff
@@ -2870,6 +2870,9 @@
                   "type": "boolean",
                   "description": "Moves the label on top of the Input Field. Default (False)."
                 },
+                "floatingLabelStyle": {
```

Comment:
add description: "the label's styles when it is in floating mode"

Reply:
Description added